zanata-war/pom.xml

```diff
@@ -163,6 +161,7 @@
         <groupId>com.ning.maven.plugins</groupId>
         <artifactId>maven-duplicate-finder-plugin</artifactId>
         <configuration>
+          <checkTestClasspath>false</checkTestClasspath>
```

Comment:
Because jersey and resteasy both have jaxrs-api. But one is in prod one is in test code.